### PR TITLE
[TFLITE]Activation functions support

### PR DESCRIPTION
**Pull request by siju-samuel**
ELU, LEAKY_RELY, LRN, LOG_SOFTMAX activation functions supported for tflite frontend.
Along with this PR, i sorted the op list of tflite.
@FrozenGene @anijain2305 @wyc-ruiker Please help to review. TIA

Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

### CODE REVIEW

**Comment by inadob** [comment]
Some of the functions already exist in another PR #4805

**Comment by siju-samuel** [comment]
@inadob thanks for letting me know., i didnt noticed, i will update my PR.

File: python/tvm/relay/frontend/tflite.py [hunk 1]
```diff
@@ -454,6 +456,45 @@ def convert_l2_normalization(self, op):
...
+        radius = lrn_options.Radius()
+        bias = lrn_options.Bias()
+        alpha = lrn_options.Alpha()
+        beta = lrn_options.Beta()
+        size = (radius * 2) + 1
+        alpha = alpha * size
+
+        # TFLite supports lrn only over the last dim
+        input_tensor_rank = len(input_tensor.tensor.ShapeAsNumpy())
+        axis = input_tensor_rank - 1
```
Comment (wyc-ruiker) [inline python/tvm/relay/frontend/tflite.py]: I observed that here is `attr_new['axis'] = 3 # Fix axis, NHWC format` in tensorflow.py. Are the two implementations consistent?

Comment (siju-samuel) [reply python/tvm/relay/frontend/tflite.py]: Yes, same, nhwc format. axis=3

Comment (wyc-ruiker) [reply python/tvm/relay/frontend/tflite.py]: > Yes, same, nhwc format. axis=3

OK, maybe we could use `axis = 3` same as tensorflow.py.

**Comment by siju-samuel** [comment]
Rebased to latest.
@FrozenGene @anijain2305 please review and help to merge. TIA

File: python/tvm/relay/frontend/tflite.py [hunk 2]
```diff
@@ -62,70 +62,72 @@ def __init__(self, model, subgraph, exp_tab):
...
+            'CAST': self.convert_cast,
+            'CEIL': self.convert_ceil,
+            'CONCATENATION': self.convert_concatenation,
+            'CONV_2D': self.convert_conv2d,
+            'COS': self.convert_cos,
+            'DEPTHWISE_CONV_2D': self.convert_depthwise_conv2d,
+            'DETECTION_POSTPROCESS': self.convert_detection_postprocess,
+            'DIV': self.convert_div,
+            'ELU': self.convert_elu,
+            'EQUAL': self.convert_equal,
```
Comment (FrozenGene) [inline python/tvm/relay/frontend/tflite.py]: Do you change it as alphabetical order?

Comment (siju-samuel) [reply python/tvm/relay/frontend/tflite.py]: @FrozenGene Yes. its better to be in alphabetical order like TF. And it reduces the conflicts while merging PR.

**Comment by FrozenGene** [comment]
Thanks, it is merged

**Pull request merged**